### ISSUE TITLE
[ISSUE] None of the drawing tools show pressure sensitivity on the canvas 

### ISSUE THREAD

**Issue by Kaikogf** [pencil2d/pencil#1303]
### Actual Results
None of the drawing tools shows pressure sensitivity
 

### System Information
+ Pencil2D Version:
Nightly Build from 01/09/2020
 

+ Operating System:
Windows8.1
 

+ Graphics Tablet:
HUION GT 185HD

**Comment by scribblemaniac** [comment]
Am I correct in assuming an issue that is new to the latest nightly build? If so, it could be caused by the recent update to the version of Qt used to build the nightlies.

**Comment by Jose-Moreno** [comment]
@Kaikogf @scribblemaniac I have proper pressure sensitivity with Jan 9th build, using a Genius MousePen 8x6.

Are you using the 32bit version? or the 64bit version? Is there any other version that does work for you?

Can you also try to restart your HUION drivers using Windows services? You can also reboot your computer with the tablet unplugged, then plug it in and enter the preferences dialog. Once you verify the drivers are loaded, then open pencil2d Jan 9th build and test again.

**Comment by scribblemaniac** [comment]
I have proper pressure sensitivity as well, tested on Windows 10 with a Huion H430P. Obviously make sure that you have the Pressure tool option checked and that you do not have mouse mode enabled in your tablet driver. Additionally, make sure that the Windows Ink option in your driver is *on* if there is such an option. Pressure sensitivity does not work for me without it, but I feel like it might have worked without it before.

**Comment by Kaikogf** [comment]
Hi to all
Very sorry for the delay.

@Jose-Moreno, as @scribblemaniac mentioned above I believe the issue was related with the last nightly build that I tested at that time (and to be honest it was kind of difficult to build the new code and there was also an update of the QtCreator...  

Now with the last build from Github (25 Feb 2020) the issue is no longer here. Closing this
@Jose-Moreno, thank you for your suggestions.
